src/hub/PolicyHub.Service/Models/PolicyFileContent.cs

```diff
@@ -39,6 +39,12 @@ public record PolicyFileContent
     [property: JsonPropertyName("permission")] Permission Permission
 );
 
+public record PolicyContext
+{
+    [JsonPropertyName("cx-policy")]
+    public required string CxPolicy { get; set; }
+}
+
```

Comment:
You can use a constructor since the cxPolicy should always be set

```suggestion
public record PolicyContext(
    [property: JsonPropertyName("cx-policy")] string CxPolicy
)

```